### ISSUE TITLE
rtl_433

### ISSUE THREAD

**Issue by zuckschwerdt** [baycom/tfrec#7]
Hi Georg & Deti,
do you know about [rtl_433](https://github.com/merbanan/rtl_433/)? Might be worthwhile to join efforts.

**Comment by petterreinholdtsen** [comment]
Yeah, rtl_433 support more and other hardware, while trec handle its specific hardware in more detail.
